src/rgw/rgw_qos.h

```diff
@@ -0,0 +1,256 @@
+#pragma once
+#include <chrono>
+#include <thread>
+#include <condition_variable>
+#include "rgw_common.h"
+
+
+class RateLimiter {
+  static constexpr int64_t fixed_float_rgw_ratelimit = 1000;
+  // counters are tracked in multiples of fixed_float_rgw_ratelimit
+  struct counters {
+    int64_t ops = 0;
+    int64_t bw = 0;
```

Comment:
this is "bytes" not "bandwidth" (=bytes/sec)

Reply:
ok